pyart/util/radar_utils.py

```diff
@@ -0,0 +1,184 @@
+""" Functions for working radar instances. """
```

Comment:
This needs a top level docstring like the other modules.  This new module will also need to be added to the dev documentation by adding a line to the doc/source/dev_reference/util.rst file.